content/posts/2023-08-24-pycon-tw-2023-sponsors.md

```diff
@@ -0,0 +1,54 @@
+Title: PyCon Taiwan 2023 贊助商行前招呼
+Date: 2023-08-24 23:00
```

Comment:
This post was not actually posted yesterday but has a past timestamp. (yep, I know you sent the PR yesterday) It might look a bit weird. I'm good with it. Just want to make sure you aware of this.

Reply:
Updated.